service/src/main/frontend/src/pages/DataDictionary.vue

```diff
@@ -0,0 +1,329 @@
+<template>
+  <div class="banner"
+    >PLACEHOLDER - BANNER</div>
```

Comment:
How do we go about customizing this at deployment time?

Reply:
I was thinking going to `dist/spa` and then the `DataDictionary.js` file and just replacing that banner name when sending it over. Do you think using ThymeLeaf is better?

Reply:
Fixed this by using a REST endpoint @ `/banner` instead.